moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -1019,11 +1040,15 @@ bool IKFastKinematicsPlugin::getPositionIK(const geometry_msgs::Pose& ik_pose, c
 
   IkSolutionList<IkReal> solutions;
   int numsol = solve(frame, vfree, solutions);
+  std::vector<std::vector<double>> solutions_obey_limits;
+  std::vector<double> dists;
+  bool solution_found = false;
```

Comment:
rather than this extra bool, why not just check ``!solutions_obey_limits.empty()``?